operators/apicast-community-operator/0.8.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,13 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: alpha,stable
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: apicast-community-operator
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.2.0
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v2
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+  com.redhat.openshift.versions: v4.6
```

Comment:
@valerymo any thoughts on this? 
I'm pretty sure we have not tested it on 4.6.
@eguzki do you know what are the requirements to this version?
IMO, it should be released on OCP version that we know it works on (4.12 +). Unless there's some other "rule" when it comes to minimum version that I'm not aware of?

Reply:
4.11 is EOL

It should be indeed 4.12

Reply:
@eguzki @MStokluska  It's done.  updated to 4.12.
Thank you
Sorry, I will be in PTO next 2 days. Could yoou please approve/merge PR if it will be ok for you. Thank you